samples/bootloader/pm.yml

```diff
@@ -13,7 +13,7 @@ s0_pad:
   share_size: mcuboot_pad
   placement:
     after: b0
-    align: {start: DT_FLASH_ERASE_BLOCK_SIZE}
+    align: {start: CONFIG_FPROTECT_BLOCK_SIZE}
```

Comment:
I don't think I understand your question. 

Note that the current file is preprocessed with b0's kconfig.

If mcuboot enables FPROTECT, then any pm configuration which uses `CONFIG_FPROTECT_BLOCK_SIZE` as its align requirement will be aligned accordingly. Also the block size used by the fprotect module will be as configured for MCUBoot.

I don't see how the app not having FPROTECT can affect any of this.

Reply:
This alignment is needed if mcuboot will use fprotect. However, the preprocessing happens based on b0's config, not mcuboot's.

Reply:
If B0 is present, and mcuboot is configured to use FPROTECT, then mcuboot will lock the are between S0 and the mcuboot_primary.

If B0 is NOT present, and mcuboot is configured to use FPROTECT, then mcuboot will lock the image partition for mcuboot. (This will be a different PR).

In the B0 - MCUBoot use case, we will always have FPROTECT enabled in MCUBoot. If for some reason the configurations mis-match you either waste space by aligning, or won't boot because of bad alignment, neither of which I consider a issue that we need to deal with.

Reply:
ping @oyvindronningstad